lib/Service/ColumnTypes/SelectionBusiness.php

```diff
@@ -51,7 +51,7 @@ public function canBeParsed($value, ?Column $column = null): bool {
 		}
 
 		$intValue = (int) $value;
-		if ("".$intValue === $value) {
+		if ((string) $intValue === (string) $value) {
```

Comment:
```suggestion
		if ((string)(int)$value === (string)$value) {
```

I guess these two lines can be unified into one line.